botocore/handlers.py

```diff
@@ -596,6 +596,56 @@ def parse_get_bucket_location(parsed, http_response, **kwargs):
     parsed['LocationConstraint'] = region
 
 
+def handle_expires_header(response_dict, customized_response_dict, **kwargs):
```

Comment:
Is there a reason why we're also not checking the operation model output shape to make sure if has the `Expires` member modeled? I may have missed that conversation

Reply:
fixing that in new commit